metron-platform/metron-common/src/main/java/org/apache/metron/common/query/PredicateProcessor.java

```diff
@@ -18,12 +18,20 @@
 
 package org.apache.metron.common.query;
 
-import org.antlr.v4.runtime.*;
-import org.apache.metron.common.query.generated.*;
+import org.antlr.v4.runtime.ANTLRInputStream;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.TokenStream;
+import org.apache.metron.common.query.generated.PredicateLexer;
+import org.apache.metron.common.query.generated.PredicateParser;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
 
 
 public class PredicateProcessor {
   public boolean parse(String rule, VariableResolver resolver) {
+    if (isEmpty(rule)) {
```

Comment:
To clarify, I've added the following test cases:
```
Assert.assertTrue(run(null, v -> variableMap.get(v)));
Assert.assertTrue(run("", v -> variableMap.get(v)));
Assert.assertTrue(run(" ", v -> variableMap.get(v)));
```
The null results in an NPE from ANTLR. 
Either we:
1. Add an else to return true 
2. Do this(my vote): if (rule == null || isEmpty(rule.trim())) { return true; }
3. Or throw a new exception for the parser